prisma/schema.prisma

```diff
@@ -69,46 +46,49 @@ model ProductFeature {
   product_id            Int
   feature_id            Int
   implementation_status ImplementationStatus
-  release_id            Int?
-  release               Release?             @relation(fields: [release_id], references: [id], onDelete: Cascade)
-  dependencies          ProductFeature[]     @relation("dependencies")
-  dependants            ProductFeature[]     @relation("dependencies")
   note                  String?              @db.VarChar(1024)
-  product               Product              @relation(fields: [product_id], references: [id], onDelete: Cascade)
+  release_id            Int?
   feature               Feature              @relation(fields: [feature_id], references: [id], onDelete: Cascade)
+  product               Product              @relation(fields: [product_id], references: [id], onDelete: Cascade)
+  release               Release?             @relation(fields: [release_id], references: [id], onDelete: Cascade)
+  product_features_B    ProductFeature[]     @relation("dependencies")
+  product_features_A    ProductFeature[]     @relation("dependencies")
 
   @@unique([product_id, feature_id])
   @@map("product_features")
 }
 
 model Release {
-  id                 Int                @id @default(autoincrement())
-  name               String
-  date               DateTime?
-  released           Boolean
-  product_features   ProductFeature[]
-  product_lifecycles ProductLifecycle[]
-  release_versions   ReleaseVersion[]
+  id                   Int                @id @default(autoincrement())
+  name                 String
+  general_availability DateTime?
+  released             Boolean
+  end_of_development   DateTime?
```

Comment:
I looked through the proposed OpenEoX schema and I don't think a conclusion has been reached yet.
Maybe for now we only want to add a field for end of life?

Also: Could you explain why we have two fields released/general availability? Not sure I understand the difference.

https://github.com/oasis-tcs/openeox/pull/13

Reply:
@soenkeliebau in case you have input

Reply:
`general_availability` says when a release has/will become generally available. This is to help communications in advance and document actions in the future.
`released` is flag that is to be turned on on the date (or not in case foreseen events).